js/src/apbct-public--2--public.js

```diff
@@ -165,6 +166,151 @@ function checkEmail(e) {
     }
 }
 
+/**
+ * @param {mixed} e
+ */
+function checkEmailExist(e) {
+    let currentEmail = e.target.value;
+    if (currentEmail && !(currentEmail in ctCheckedEmailsExist)) {
+        viewCheckEmailExist(e, 'load');
+        // Using REST API handler
+        if ( ctPublicFunctions.data__ajax_type === 'rest' ) {
+            apbct_public_sendREST(
+                'check_email_exist_post',
+                {
+                    method: 'POST',
+                    data: {'email': currentEmail},
+                    callback: function(result) {
```

Comment:
This callback function is duplicated into the common ajax handler too. Please, extract it.

Reply:
Fixed in new commits